docs/app/controllers/demo.js

```diff
@@ -1,6 +1,8 @@
-import { inject as service } from '@ember/service';
 import Controller from '@ember/controller';
 import { isEmpty } from '@ember/utils';
+import * as emberService from '@ember/service';
+
+const service = emberService.service ?? emberService.inject;
```

Comment:
This isn't needed. Ember Bootstrap requires Ember 4.8 or above. The named import `service` is available since Ember 4.1. We can simplify use the new import.